providers/zendesk.go

```diff
@@ -1,11 +1,11 @@
 package providers
 
 const (
-	ZendeskChat    Provider = "zendeskChat"
+	ZendeskChat    Provider = "zendeskSupport"
```

Comment:
This seems like a bug. `ZendeskChat` and `ZendeskChat` both map to the same value.

Reply:
Likely when I was copy pasting I did this crime :hot_face: